### PR TITLE
PSFramework

### PR DESCRIPTION
**Pull request by FriedrichWeinmann**
Added new module

### CODE REVIEW

File: Modules/PSFramework.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Name: PSframework
+
+## Description:
+Module designed to provide powerful infrastructure features to powershell scripters. Whether it is to provide features, save time, improve user experience or drive manageability of PowerShell itself.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PowerShell | Friedrich Weinmann (<a href="http://allthingspowershell.blogspot.de" target="_blank">b</a> \| <a href="https://twitter.com/FredWeinmann" target="_blank">t</a>) | [Github](https://github.com/PowershellFrameworkCollective/psframework) \| [official project site](https://github.com/PowershellFrameworkCollective/psframework) | 4 / 2017 | Tools, Infrastructure, Framework, PowerShell, Logging, Management |
```
Comment (ClaudioESSilva) [inline Modules/PSFramework.md]: Please leave just GitHub entry for the Project URL(s) once they are the same.
Set the Creation Date as "April 2017"

File: Modules/PSFramework.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+____
+### Shared blog posts:
```
Comment (ClaudioESSilva) [inline Modules/PSFramework.md]: Please order the list alphabetically

**Review by ClaudioESSilva — CHANGES_REQUESTED** [review]
Minor changes

**Comment by FriedrichWeinmann** [comment]
done

**Review by ClaudioESSilva — APPROVED** [review]
Thanks!

**Comment by ClaudioESSilva** [comment]
Thanks for your contribution @FriedrichWeinmann ! 👍

**Pull request merged**